src/Type/Constant/ConstantArrayType.php

```diff
@@ -58,7 +61,7 @@ class ConstantArrayType extends ArrayType implements ConstantType
 	 * @param array<int, ConstantIntegerType|ConstantStringType> $keyTypes
 	 * @param array<int, Type> $valueTypes
 	 * @param int $nextAutoIndex
-	 * @param int[] $optionalKeys
+	 * @param list<int> $optionalKeys
```

Comment:
I guess this is _technically_ a BC break, but I can't see why anybody would have passed anything other than a list here.